src/main/java/uk/gov/hmcts/dts/fact/entity/Audit.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.dts.fact.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "admin_audit")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Audit {
+    @Id
+    @SequenceGenerator(name = "seq-gen", sequenceName = "admin_audit_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq-gen")
+    private Integer id;
+    @Column(name = "user_email")
+    private String userEmail;
+    @OneToOne()
+    @JoinColumn(name = "action_id")
+    private AuditType auditType;
+    @Column(name = "action_data_before")
+    private String actionDataBefore;
+    @Column(name = "action_data_after")
+    private String actionDataAfter;
+    private String location;
+    @Column(name = "creation_time")
+    private LocalDateTime creationTime;
```

Comment:
Is this the time of the entity change or the audit being created?

Reply:
audit being created